resources/views/projects/details.blade.php

```diff
@@ -90,7 +90,74 @@
 					@endforeach
 				</div>
 			</div>
+			<!-- Progress section -->
+			<div class="sixteen wide column">
+				<div class="ui left aligned detail-container">
+					<p><strong>Progress</strong></p>
+					<div class="ui blue progress">
+						<div class="bar">
+							<div class="progress"></div>
+						</div>
+						<div class="label">Currently on: Design</div>
+					</div>
+					<div class="ui right floated button" onclick="displayMilestones()">
```

Comment:
`div`s are not `button`s, in any any case you would bind the event listener to the `a` tag. The `div` only serves layout purposes